<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>muproMain</class>
 <widget class="QMainWindow" name="muproMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>500</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>muproMain</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{background-color:rgb(255, 255, 255)}

QPushButton{
	background-color:rgb(255,255,255);
	border:none;
	text-align:left;
	padding-left:20px;
	color:rgb(47,47,47);
	font: 25 15pt &quot;Helvetica Neue&quot;;
}

QPushButton:hover{
	background-color:rgb(215,232,255);
	color:rgb(54,131,249);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="button_ferroelectric">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>245</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>15</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Ferroelectric</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_metal">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>245</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>15</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Precipitation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ferromagnetic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>245</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>15</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Ferromagnetic</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_effect">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>245</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>15</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Effective property</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_metal_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>245</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>15</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Grain growth</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_metal_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>245</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>15</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Defect transport</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_metal_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>245</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>15</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Solidification</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo_LB">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>650</width>
      <height>121</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>650</width>
      <height>121</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="mupro-resource.qrc">:/img/small-logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="contact_info_LB">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:none;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>395</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item row="1" column="1">
      <widget class="QLabel" name="featureImg2_LB">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>190</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>190</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="mupro-resource.qrc">:/img/magnetic-composite.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="featureImg1_LB">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>190</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>190</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="mupro-resource.qrc">:/img/interface.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="brief_intro_LB">
       <property name="maximumSize">
        <size>
         <width>390</width>
         <height>170</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-right:5px;
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp; µ-pro, microstructure-property modeling package, is a series of phase-field models solved by spectral method and implemented with modern fortran along with MPI, thus gaining native numerical and performance advantages. It excels in dealing with numerous types of problems, ranging from grain growth, precipitation in alloys to domain structure evolution in ferroelectric/ferromagnetic/ferroelastic system.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>button_ferroelectric</zorder>
   <zorder>button_metal</zorder>
   <zorder>button_ferromagnetic</zorder>
   <zorder>button_effect</zorder>
   <zorder>button_metal_2</zorder>
   <zorder>button_metal_3</zorder>
   <zorder>button_metal_4</zorder>
   <zorder>logo_LB</zorder>
   <zorder>contact_info_LB</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mupro-resource.qrc"/>
 </resources>
 <connections/>
</ui>
